config.example.yaml

```diff
@@ -0,0 +1,10 @@
+models_dir: /models
+model_family: gptq_llama
+setup_params:
+  repo_id: anon8231489123/gpt4-x-alpaca-13b-native-4bit-128g # TheBloke/gpt4-alpaca-lora-30B-GPTQ-4bit-128g # anon8231489123/vicuna-13b-GPTQ-4bit-128g
```

Comment:
Could you please remove any links to any repos so that we don't have to deal with any licensing issues?

Reply:
Deleted the file, with no repo links it's the same as the existing `config.yaml`